src/org/daisy/braille/utils/pef/PEFGenerator.java

```diff
@@ -265,10 +265,8 @@ public void generateTestBook(File output) throws FileNotFoundException {
      * @throws FileNotFoundException If the given file object does not denote an existing,
      *                               writable regular file and a new regular file of that name cannot be created, or if
      *                               some other error occurs while opening or creating the file
-     * @throws Exception             if an error occurs
```

Comment:
I think it would be a good idea to mention here that an exception can occur if the page is not big enough to render the test image on it.